### PR TITLE
Task 11

### PR DESCRIPTION
**Pull request by evTyurin**

### CODE REVIEW

**Review by mrseneca — CHANGES_REQUESTED** [review]
It is also necessary to implement working with the database using jdbc. All logic must move from in memory storage to the database

**Review by mrseneca — CHANGES_REQUESTED** [review]
Well done. But subtask 4 says what you also should implement transactional support for your methods, but I can't find it. if i'm wrong, please comment with examples. If not please add implementation

File: server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java [hunk 1]
```diff
@@ -27,17 +39,465 @@ public void setInstance() {
...
+                } catch (SQLException exception) {
+                    exception.printStackTrace();
+                }
+            });
+            connection.commit();
+            connection.setAutoCommit(true);
+        } catch (Exception exception) {
+            throw new Exception("Can't add order");
+        }
+    }
```
Comment (evTyurin) [inline server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java:73]: Thanks. The only one method use transactional support

Comment (mrseneca) [reply server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java:73]: Is it only one place you need transaction management?

Comment (evTyurin) [reply server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java:73]: Yes, in other situations I need to update only one table per time

Comment (mrseneca) [reply server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java:73]: Ok. Good then, go ahead!

**Review by mrseneca — APPROVED** [review]

File: server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java (hunk 1 above)
Comment (evTyurin) [reply server/src/main/java/com/senlainc/warsaw/tyurin/dao/OrderDAO.java:73]: Thank you!

**Pull request merged**